### PR TITLE
sign up country selection settings added

### PR DESCRIPTION
**Pull request by muammertop**

### CODE REVIEW

File: addons/default/visiosoft/base-theme/resources/config/settings/sections.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
                 'fields' => [
                     'home_page_sub_categories_limit', 'navigation_title', 'navigation_action', 'date_fields',
                     'price_fields', 'breadcrumbs', 'ad_details', 'ad_details_tab', 'latest_and_view_all_btn',
-                    'register_page_instruction_logo', 'register_page_alert_link'
+                    'register_page_instruction_logo', 'register_page_alert_link', 'enabled_countries'
```
Comment (fatihalp) [inline addons/default/visiosoft/base-theme/resources/config/settings/sections.php:12]: we can say default country for register

Comment (muammertop) [reply addons/default/visiosoft/base-theme/resources/config/settings/sections.php]: https://github.com/openclassify/openclassify/pull/715/commits/bd153541f906be1f2ca99f2bc06675828c464928

**Review by fatihalp — APPROVED** [review]

File: addons/default/visiosoft/base-theme/resources/config/settings/settings.php [hunk 2]
```diff
@@ -71,4 +71,24 @@
...
+		'config' => [
+			'default_value' => function () {
+				return [config('visiosoft.theme.base::countries.default')];
+			},
+			'options' => function () {
+				$array = CountryModel::query()->get();
+				$cur = array();
+				foreach ($array as $key => $value) {
+					$cur[$value['abv']] = $value['name'];
+				}
```
Comment (diashalabi) [inline addons/default/visiosoft/base-theme/resources/config/settings/settings.php:89]: why not use pluck here?

Comment (muammertop) [reply addons/default/visiosoft/base-theme/resources/config/settings/settings.php]: https://github.com/openclassify/openclassify/pull/715/commits/3220976fd2b79f320c2cf9ec5eab1bc6adc01ca6

**Review by diashalabi — APPROVED** [review]

**Pull request merged**